TGC/localisation/00_decisions.csv

```diff
@@ -1856,16 +1858,16 @@ lie_purchase_alaska_desc;Citing deteriorating relations with Britian and a fanat
 lie_purchase_alaska_title;Purchase Alaska;;;;;;;;;;;;;x
 likin_taxation_desc;The likin or lijin was a form of internal tariff in the Qing Empire and Republic of China, which was first introduced as a means of financing the largely locally recruited armies to suppress the Taiping Rebellion.;;;;;;;;;;;;;x
 likin_taxation_title;Lijin Taxation;;;;;;;;;;;;;x
-limit_austrian_influence_within_army_desc;The Austrian General staff has been quite uppity to our leadership, we should unite our staffs and reshuffle the war cabinet to get rid of "old guards" that constanly compilicate strategy forming, planning, and execution.;;;;;;;;;;;;;x
+limit_austrian_influence_within_army_desc;"The Austrian General staff has been quite uppity to our leadership, we should unite our staffs and reshuffle the war cabinet to get rid of ""old guards"" that constanly compilicate strategy forming, planning, and execution.";;;;;;;;;;;;;x
```

Comment:
pls no I spent so much time trying to remove unnecessary quotation marks from localization files